exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/TestKit.java

```diff
@@ -138,7 +166,103 @@ public static TestKit forService(Class<? extends ServiceModule> serviceModule) {
     return serviceClass.cast(service);
   }
 
-  private native long nativeCreateTestKit(UserServiceAdapter[] services, boolean auditor,
+  /**
+   * Creates a block with the given transaction. In-pool transactions will be ignored.
+   *
+   * @return created block
+   * @see <a href="https://exonum.com/doc/version/0.10/advanced/consensus/specification/#pool-of-unconfirmed-transactions">Pool of Unconfirmed Transactions</a>
+   */
+  public Block createBlockWithTransaction(TransactionMessage transaction) {
+    return createBlockWithTransactions(transaction);
+  }
+
+  /**
+   * Creates a block with the given transaction(s). In-pool transactions will be ignored.
+   *
+   * @return created block
+   * @see <a href="https://exonum.com/doc/version/0.10/advanced/consensus/specification/#pool-of-unconfirmed-transactions">Pool of Unconfirmed Transactions</a>
+   */
+  public Block createBlockWithTransactions(TransactionMessage transaction,
```

Comment:
Yes, I believe it's better to distinguish with/without in-pool methods. Especially since we might want to commit an empty block without in-pool transactions - `createBlockWithTransactions()`

Reply:
I think we must first determine why would anyone want to ignore the in-pool transactions (basically, going around how Exonum works). If there is a good reason for that — _then_ I'd distringuish such classes of methods, and, probably, with a more verbose name (e.g., `commitBlock` vs `commitBlockWithOnlyTransactions`).